cc.arduino.IDE2.json

```diff
@@ -0,0 +1,117 @@
+{
+    "app-id": "cc.arduino.IDE2",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "21.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "arduino-ide",
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=ipc",
+        "--share=network",
+        "--device=all",
+        "--filesystem=home",
+        "--filesystem=xdg-run/keyring",
+        "--system-talk-name=org.gtk.vfs.Daemon",
```

Comment:
mmm.

gvfs is on the session bus and in that case we just add `--talk-name=org.gtk.vfs.*`.

Reply:
ok i will update it to that.

thanks for the help and patients with this.

Reply:
however i am going to leave it on the system bus because otherwise the application complains about not being able to use the system bus.
`[3:0527/193631.191436:ERROR:bus.cc(392)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory`